platforms/jvm/testing-jvm/src/main/java/org/gradle/api/tasks/testing/TestReport.java

```diff
@@ -53,6 +54,11 @@ public abstract class TestReport extends DefaultTask {
     private final DirectoryProperty destinationDir = getObjectFactory().directoryProperty();
     private final ConfigurableFileCollection resultDirs = getObjectFactory().fileCollection();
 
+    @Inject
+    protected BuildOperationRunner getBuildOperationRunner() {
+        throw new UnsupportedOperationException();
```

Comment:
💅  Can we have this method as abstract without method body?

Reply:
Sure.

Reply:
I ended up reverting this to match the other injections in the same class. We should change all of them to `abstract` in a single PR perhaps, maybe do that in 9.0 to avoid breaking anyone?

Reply:
I added an issue for 9.0: https://github.com/gradle/gradle/issues/28435